src/Files.App.Controls/Files.App.Controls.csproj

```diff
@@ -15,6 +15,33 @@
         <Optimize Condition="'$(Configuration)' != 'Debug'">true</Optimize>
     </PropertyGroup>
 
+	<PropertyGroup>
+		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<Title>Files UI Controls</Title>
+		<Authors>yaira, files-community, mdta</Authors>
```

Comment:
```suggestion
		<Authors>files-community</Authors>
```

Reply:
I'm not against it if @mdtauk would like the callout, but we don't generally list individual authors.